react/react1/week2/react-week2/src/components/TodoTimer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+
+export default function Timer() {
+const [seconds, setSeconds] = React.useState(1);
+
+
+ React.useEffect(() => {
+   if (seconds > 0) {
```

Comment:
You don't need this `if` condition, since the initial value is `1` and it's always going to be 1 when the app is started, and the user has no way to change that number, therefore this is not necessary.

Reply:
ok